extensions/lifecycle/LifecycleQueuePopulator.js

```diff
@@ -225,7 +225,7 @@ class LifecycleQueuePopulator extends QueuePopulatorExtension {
         const value = JSON.parse(entry.value);
 
         const operation = value.originOp;
-        if (operation !== 's3:ObjectRestore') {
+        if (!['s3:ObjectRestore', 's3:ObjectRestore:Post'].includes(operation)) {
```

Comment:
a comment explaining that this s3:ObjectRestore value is used for backwards compatibility with older cloudserver would be nice